pipeline/cvp/Jenkinsfile-cvp.groovy

```diff
@@ -0,0 +1,108 @@
+/*
+    Pipeline script for executing CVP test suites
+*/
+// Global variables section
+
+def nodeName = "centos-7"
+def sharedLib
+def jobStatus
+
+def sendCVPUMBMessage(def ciMsg, def status) {
+    /*
+        Trigger a UMB message for successful cvp test completion
+    */
+    def msgMap = [
+        "category": "RHCEPH CVP",
+         "status": "${status}",
```

Comment:
whitespace problem here

It would be great to run all these Groovy scripts through a linter like `npm-groovy-lint`